Upstream-Status: Backport

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From a3366323b2070531a8df34b826b89cca8f1557de Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppymaster@gmail.com>
Date: Wed, 15 Nov 2017 01:47:20 +0000
Subject: [PATCH] Fix build with gcc5
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Resolves this build failure with gcc5.

In file included from ../../third_party/WebKit/Source/core/exported/LocalFrameClientImpl.cpp:93:0:
../../third_party/WebKit/public/web/WebFrameClient.h: In member function ‘virtual blink::WebString blink::WebFrameClient::GetInstrumentationToken()’:
../../third_party/WebKit/public/web/WebFrameClient.h:510:56: error: could not convert ‘(const char*)""’ from ‘const char*’ to ‘blink::WebString’
   virtual WebString GetInstrumentationToken() { return ""; }

Change-Id: Ibd666de9020ae14ca695277d73ee451362e1522f
Reviewed-on: https://chromium-review.googlesource.com/765570
Reviewed-by: Pavel Feldman <pfeldman@chromium.org>
Reviewed-by: Kinuko Yasuda <kinuko@chromium.org>
Commit-Queue: Kinuko Yasuda <kinuko@chromium.org>
Cr-Commit-Position: refs/heads/master@{#516554}
---
 third_party/WebKit/public/web/WebFrameClient.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/WebKit/public/web/WebFrameClient.h b/third_party/WebKit/public/web/WebFrameClient.h
index ac14deaff897..c2e1ec3d0a25 100644
--- a/third_party/WebKit/public/web/WebFrameClient.h
+++ b/third_party/WebKit/public/web/WebFrameClient.h
@@ -509,7 +509,7 @@ class BLINK_EXPORT WebFrameClient {
   // Returns string to be used as a frame id in the devtools protocol.
   // It is derived from the content's devtools_frame_token, is
   // defined by the browser and passed into Blink upon frame creation.
-  virtual WebString GetInstrumentationToken() { return ""; }
+  virtual WebString GetInstrumentationToken() { return WebString(); }
 
   // PlzNavigate
   // Called to abort a navigation that is being handled by the browser process.
-- 
2.13.6

